### PR TITLE
CMR-4945: UMM-C default version 1.9 -> 1.10

### PR DESCRIPTION
**Pull request by mschmele**
NOTE: I expect some tests to fail here. I'll get to them once CI finishes

### CODE REVIEW

File: system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj [hunk 1]
```diff
@@ -492,15 +492,19 @@
           ;; Check that each concept was updated
           (doseq [concept-id concept-ids
                   :let [concept (-> (search/find-concepts-umm-json :collection
-                                                                   {:concept-id concept-id})
+                                                                   {:concept-id (first concept-ids)})
```
Comment (sxu123) [inline system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj]: Why the change?  We want to test each concept-id in concept-ids, now every iteration it's testing the same first concept-id?

File: system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj [hunk 2]
```diff
@@ -573,7 +577,7 @@
...
```
Comment (ygliuvt) [inline system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj]: This line should not have been changed.

Comment (mschmele) [reply system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj]: You are correct 😄 - I had changed it for debugging and forgot to undo that

**Comment by sxu123** [comment]
The empty :Instruments is added because for version 1.10, it doesn't go through umm migration to have the it removed. Ideally, it probably should be added in the migration code when source=dest. But since MMT is depending on this, we could revisit in the future.
